src/screens/components/TokenOperations/index.tsx

```diff
@@ -114,6 +116,9 @@ const TokenOperations = ({
         }
       }
       await waiter();
+       setTimeout(() => {
```

Comment:
do you need the 7 sec timeout?

Reply:
Yes, I talked to Doron yesterday and he told me that this is the optimal time needed